### ISSUE TITLE
Handle content from screen reader users that makes no sense out of context.

### ISSUE THREAD

**Issue by jasonwebb** [jasonwebb/supertunnel#10]
Some content in the app is reachable with a screen reader, but lacks context in the DOM to make it understandable. For example, the floating text labels on the canvas only make sense when you see that they are attached to specific locations on the globe. However, the globe is rendered in the `<canvas>` element, and is not exposed in any way to screen reader users, so they end up just seeing a piece of text in the DOM that has no context.

We can handle this in two ways:

1. **Add** context to these text elements using sr-only text, or
2. **Hide** this content from screen readers and replicate it elsewhere in the app. Most of this content (like the origin and destination points) already exist in `<h2>`s, but some doesn't (like the distance).
